README.md

```diff
@@ -1,3 +1,43 @@
-# Castone Project
+# Capstone Project
 
-This is the capstone project github repo
+This is the first capstone project github repo
+
+> In this is a website designed using css, javascript and html based on this design <https://www.behance.net/gallery/29845175/CC-Global-Summit-2015>
+
+## Built With
+
+- Eslint
+- Webhint
+- Styelint
+- CSS/Sass
+
+## Live Demo
+
+[https://mrbrn197.github.io/Capstone-Project-One/](https://mrbrn197.github.io/Capstone-Project-One/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+`git clone https://github.com/MrBrN197/Capstone-Project-One.git`
+
+## Authors
+
+👤 **Author1**
```

Comment:
- [ ] OPTIONAL
Kindly replace the text `**Author1**` with your full name, so as to make your repo more professional

Reply:
Thank You. I've changed this.